Api/Data/ImportInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Itonomy\Katanapim\Api\Data;
+
+interface ImportInterface
+{
+    public const PRODUCT_IMPORT_JOB_CODE = 'itonomy_katanapim_products_import';
+    public const SPECIFICATIONS_IMPORT_JOB_CODE = 'itonomy_katanapim_specifications_import';
+    public const SPECIFICATIONS_GROUP_IMPORT_JOB_CODE = 'itonomy_katanapim_specifications_group_import';
```

Comment:
I think should move these to the import instances and name them all the same, like `IMPORT_JOB_CODE`